README.md

```diff
@@ -1,24 +1,35 @@
-<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
Looks like you accidentally removed the pre-commit marker. Did you generate this by hand?

Reply:
This was generated with terraform-docs (` terraform-docs md ./ > README.md`) and I'm not sure why this was removed. Will add back.

Reply:
Ah. That's what I meant by "generated by hand". Typically we just let the pre-commit hook do it for us. That's what the markers are for.

Reply:
added back